composer.json

```diff
@@ -30,23 +30,23 @@
         }
     },
     "require-dev": {
-        "symfony/dom-crawler": "~3",
+        "symfony/dom-crawler": "~4 || ~5",
```

Comment:
Decide for one Symfony version constraint, so either all 4.x versions or 4.4

Reply:
True! Updated.